public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_fields.tsx

```diff
@@ -55,10 +55,10 @@ export const DataConfigPanelFields = ({
     if (visType === VIS_CHART_TYPES.HeatMap)
       return name === AGGREGATIONS ? list.length >= 1 : list.length >= 2;
     // condition for line and scatter for dimensions section.
-    return name === GROUPBY && (list.length >= 1 || !isEmpty(time_field));
+    return name === GROUPBY && (list.length >= 1 || !isEmpty(timeField));
   };
 
-  const { time_field, unit, interval } = dimensionSpan;
```

Comment:
Again why was this field name changed?

Reply:
Same linting fix, rules only accept camelCase, CamelCase, or CAMEL_CASE